packages/client/screens/MySettings/settingsMachine.ts

```diff
@@ -323,6 +342,114 @@ export const settingsMachine =
                                 },
                             },
                         },
+                        'Link Google account manager': {
+                            initial: 'Has user already linked a google account',
+
+                            states: {
+                                'Has user already linked a google account': {
+                                    always: [
+                                        {
+                                            cond: 'User has already linked a google account',
+                                            target: 'User has linked a google account successfully',
+                                        },
+                                        {
+                                            target: 'Waiting for user to start a link google account operation',
+                                        },
+                                    ],
+                                },
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏I like this choose state!